tests/test_coastal_vulnerability.py

```diff
@@ -1428,6 +1428,26 @@ def test_missing_sealevelrise_field(self):
                 if 'Value must be one of:' in err_strings:
                     raise ValueError(err_strings)
 
+    def test_check_slr_geometry_type(self):
+        """CV Validate: test catching non-point geometries in SLR input"""
+        gpkg_driver = ogr.GetDriverByName("GPKG")
+        # create a non-point/multipoint geometry
+        polygon_path = os.path.join(self.workspace_dir, 'slr_poly.gpkg')
+        polygon_vector = gpkg_driver.CreateDataSource(polygon_path)
+        srs = osr.SpatialReference()
+        srs.ImportFromEPSG(4326)
+        polygon_vector.CreateLayer('layer', srs, ogr.wkbPolygon)
+        polygon_vector = None
+
+        args = {'slr_vector_path': polygon_path}
+        warning_messages = coastal_vulnerability.validate(args)
+
+        with self.assertRaises(ValueError):
+            for keys, message in warning_messages:
+                print(keys, message)

```

Comment:
Probably don't need to keep this `print`, right?

Reply:
ahh thanks for catching that!